### DISCUSSION TITLE
Popover content react rendering.

### DISCUSSION THREAD

**Discussion by DomainFlag** [twbs/bootstrap#34562] [General]
Is it possible to render react components as content for the popover?

**Comment by RyanBerliner** [comment]
I've created a library to use bootstrap's javascript nicely alongside react -- and recently released support for this. There is 1 caveat that the popover will not animate (no fading in and out).

[View a demo of popover renderTitle and renderContent props in action](https://ryanberliner.com/bs5-react-elements/demos/popover-render-content/)
[https://github.com/RyanBerliner/bs5-react-elements](https://github.com/RyanBerliner/bs5-react-elements)

This may work for your use case, otherwise you can view the source and implement something similar yourself, or you'd probably want to use one of the react rewrites of bootstrap.
